### PR TITLE
RT-2.12 Fixed with deviations & minor changes

### PR DESCRIPTION
**Pull request by snaragund**
- Defined new Nokia deviation 'EnableTableConnections' & updated metadata.proto, metadata.pb.go & deviations.go.
- Added deviations required.
- Edited dp2v6Route such that it is advertised as ISIS route.
- Set separate tag value for IPv4 & IPv6 as per README.
- Added missing 'metricPropogation' parameter in 2.12.8 case.

PS: Script needs alteration (apart from this PR) such that it adheres to README.

"This code is a Contribution to the OpenConfig Feature Profiles project ("Work") made under the Google Software Grant and Corporate Contributor License Agreement ("CLA") and governed by the Apache License 2.0. No other rights or licenses in or to any of Nokia's intellectual property are granted for any other purpose. This code is provided on an "as is" basis without any warranties of any kind."

### CODE REVIEW

File: feature/isis/otg_tests/static_route_isis_redistribution/static_route_isis_redistribution_test.go [hunk 1]
```diff
@@ -366,6 +375,46 @@ func getTagSetName(dut *ondatra.DUTDevice, policyName, stmtName, afStr string) s
...
+				},
+			},
+		}
+
+		gnmiClient := d.RawAPIs().GNMI(t)
+		if _, err := gnmiClient.Set(context.Background(), gpbSetRequest); err != nil {
+			t.Fatalf("Unexpected error updating SRL static-route tag-set: %v", err)
+		}
+	default:
+		t.Fatalf("Unsupported vendor %s for deviation 'UseVendorNativeACLConfiguration'", d.Vendor())
```
Comment (dplore) [inline feature/isis/otg_tests/static_route_isis_redistribution/static_route_isis_redistribution_test.go:414]:
```suggestion
		t.Fatalf("Unsupported vendor %s for deviation 'EnableTableConnections'", d.Vendor())
```

File: feature/isis/otg_tests/static_route_isis_redistribution/static_route_isis_redistribution_test.go [hunk 2]
```diff
@@ -113,7 +114,7 @@ func getAndVerifyIsisImportPolicy(t *testing.T,
 	if err != nil {
 		t.Fatalf("failed due to %v", err)
 	}
-	t.Log(getResponse)
+	t.Log("Recieved parameters of table connections")
```
Comment (dplore) [inline feature/isis/otg_tests/static_route_isis_redistribution/static_route_isis_redistribution_test.go:117]:
```suggestion
	t.Log("Received parameters of table connections")
```

**Review by dplore — CHANGES_REQUESTED** [review]
minor nit comments

**Comment by dplore** [comment]
Please also check that [fptest virtual](https://github.com/openconfig/featureprofiles/pull/3513#issuecomment-2407811147) test passes on your platform as expected

File: feature/isis/otg_tests/static_route_isis_redistribution/static_route_isis_redistribution_test.go (hunk 1 above)
Comment (snaragund) [reply feature/isis/otg_tests/static_route_isis_redistribution/static_route_isis_redistribution_test.go]: Updated as per comments.

**Comment by snaragund** [comment]
Shared hardware logs through https://partnerissuetracker.corp.google.com/issues/380289415 please do check.

**Review by dplore — APPROVED** [review]

**Pull request merged**